src/Containers.jsx

```diff
@@ -627,8 +627,10 @@ class Containers extends React.Component {
             if (this.props.textFilter.length > 0) {
                 const lcf = this.props.textFilter.toLowerCase();
                 filtered = filtered.filter(id => this.props.containers[id].Name.toLowerCase().indexOf(lcf) >= 0 ||
+                    this.props.containers[id].Config?.Labels?.PODMAN_SYSTEMD_UNIT.toLowerCase().indexOf(lcf) >= 0 ||
                     (this.props.containers[id].Pod &&
-                     this.props.pods[utils.makeKey(this.props.containers[id].uid, this.props.containers[id].Pod)].Name.toLowerCase().indexOf(lcf) >= 0) ||
+                     (this.props.pods[utils.makeKey(this.props.containers[id].uid, this.props.containers[id].Pod)].Name.toLowerCase().indexOf(lcf) >= 0 ||
+                     this.props.pods[utils.makeKey(this.props.containers[id].uid, this.props.containers[id].Pod)].Labels?.PODMAN_SYSTEMD_UNIT?.toLowerCase().indexOf(lcf) >= 0)) ||
```

Comment:
these are quite long lines, I think you should put the `this.props.pods[utils.makeKey(this.props.containers[id].uid, this.props.containers[id].Pod)]` into a variable as it is same for both

Reply:
Uhhh, sure but that means expanding the filter :) We can't put it in `lcf` as it is in a loop